package.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "notificationlib",
+    "version": "1.0.0",
+    "description": "User interface for the OCTRI Notification Library",
+    "main": "index.js",
+    "scripts": {
+        "analyze": "webpack --mode=production; webpack --profile --json > bundle-stats.json; webpack-bundle-analyzer bundle-stats.json ./target/classes/static/assets",
+        "build": "webpack --mode=production",
+        "build-css": "sass  --load-path ./node_modules/ src/main/resources/frontend/scss/app.scss ./target/classes/static/assets/css/app.css",
+        "build:dev": "webpack --mode=development",
+        "format": "prettier --write ./*.js src/main/resources/**/*.{js,vue,spec.js}",
+        "format:check": "prettier --check ./*.js src/main/resources/**/*.{js,vue,spec.js}",
+        "lint:js": "eslint ./*.js src/main/resources/**/*.{js,vue,spec.js}",
```

Comment:
If you haven't already, you should copy the `.eslintrc.js` and `.prettierrc.js` files into this project.